src/WebJobs.Script.Grpc/Server/DefaultHttpProxyService.cs

```diff
@@ -97,7 +97,12 @@ public void StartForwarding(ScriptInvocationContext context, Uri httpUri)
             httpContext.Items[ScriptConstants.HttpProxyingEnabled] = bool.TrueString;
 
             // add invocation id as correlation id
-            httpRequest.Headers.TryAdd(ScriptConstants.HttpProxyCorrelationHeader, context.ExecutionContext.InvocationId.ToString());
+            if (!httpRequest.Headers.TryAdd(ScriptConstants.HttpProxyCorrelationHeader, context.ExecutionContext.InvocationId.ToString()))
```

Comment:
Rather than call `ToString()` twice -- call it once and use that later. In general, we'll need to sweep through and clean all of these `ToString()` calls up in the future, but let's not add to them.